app/javascript/widget/components/ContinueChatButton.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <button
+    type="button"
+    class="flex w-full justify-between items-center rounded-md ring-1 ring-inset ring-slate-50 bg-white px-2 py-2 text-sm text-slate-700 shadow-sm hover:bg-slate-25 dark:text-white dark:bg-slate-800 dark:ring-slate-900 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+    @click="$emit('continue')"
+  >
+    <div
+      class="w-10 h-10 rounded-md bg-slate-75 dark:bg-slate-700 text-lg flex items-center justify-center flex-shrink-0"
+    >
+      💬
+    </div>
+    <div class="text-left flex flex-col justify-start flex-grow mx-2">
+      <h5 class="font-medium text-slate-900 text-brand-link dark:text-white">
+        {{ title }}
+      </h5>
+      <p class="h-4 leading-4">
+        <span
+          v-if="unreadCount > 0"
+          class="inline-flex items-center rounded-full bg-green-200 px-1 min-w-[16px] leading-4 text-xs font-medium text-green-700 mr-0.5"
+        >
+          {{ unreadCount }}
+        </span>
+        <span
+          v-dompurify-html="content"
+          class="leading-4 h-4 text-ellipsis overflow-hidden dark:text-slate-25"
```

Comment:
@nithindavid If the last message is more extended. Then there is an overflow.

<img width="401" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/376ed5ff-d561-4945-822e-5ccf9a357e0f">

Reply:
If the last message is an attachment, then there is a space. We need to handle images, audio, and file attachments. The same is the case for outgoing and incoming.

<img width="401" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/5eeec721-2904-4be7-8262-d430018d56dc">